pkgs/development/node-packages/node-packages.json

```diff
@@ -194,6 +196,9 @@
 , "postcss"
 , "postcss-cli"
 , "prettier"
+, "prettier_d_slim"
```

Comment:
This should work fine as it's a command line tool.

```console
❯ nix-build -A 'nodePackages.prettier_d_slim'
❯ result/bin/prettier_d_slim status
Not running
```

Reply:
@midchildan thank you so much for the comment, I will remove those packages and do it again.